swagger.yaml

```diff
@@ -5175,6 +5175,8 @@ definitions:
           type: string
       speed_mbps:
         type: integer
+      interface_type:
```

Comment:
Maybe just `type`?

`interface.interface_type` seems not great.

@avishayt what do you think?

Reply:
I agree, it's under "interface", so the "interface_" prefix is redundant.

Reply:
Update: used `type` and didn't run into any issues!